docs/DeveloperGuide.md

```diff
@@ -807,53 +826,48 @@ Priorities: 1 (must have), 2 (nice to have), 3 (unlikely to have)
 * 2a. tCheck detects an incorrect input format.
 
         * 2a1. tCheck requests for the input to be in the correct format.
-        
+
       	* 2a2. User enters new data.
-      	
+
       	Steps 2a1-2a2 are repeated until the data entered are correct.
-      	
+
       	Use case resumes from step 3.
-      	
+
 * 2b. tCheck detects an empty contact list.
 
    	    * 2b1. tCheck shows a warning message.
-   	    
+
    	    Use case ends.
 
 **UC03 - Set ingredient level for a single ingredient**
 
 **MSS**
 
 1. User chooses to set the ingredient level for an ingredient.
-2. tCheck requests for the name of the ingredient.
-3. User enters the name of the ingredient.
-4. tCheck requests for the amount of the particular ingredient.
-5. User enters the amount of the particular ingredient.
-6. tCheck will set the ingredient level for this ingredient and displays a success message.
+2. User enters the name of the ingredient and the amount he/she wants to set to.
+3. tCheck will set the ingredient level of this ingredient and displays a success message.
```

Comment:
Should be "display" instead of "displays", because "will ... display"

Reply:
yea, will change